### PR TITLE
Disable automatic conversion of line ending in files

### PR DESCRIPTION
**Pull request by malkia**
Fixes integrity checksums on text files, where due to line end conversion, the file may end up with a different checksum than intended (for example on Windows).

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text eol=lf
```
Comment (fmeum) [inline .gitattributes:1]:
```suggestion
* -crlf
```
is closer to what we want as it just disables any automatic conversion without forcing a particular type of line ending.

**Review by malkia — COMMENTED** [review]
LGTM

**Comment by fmeum** [comment]
@malkia Could you update the PR title and description as they will become the commit message?

**Comment by malkia** [comment]
@fmeum Let me know if I did better. I'm still n00b when comes to `PR`'s - did  few in the past for different projects. Thanks for the feedback!

**Review by fmeum — APPROVED** [review]
Looks great, thanks!

**Pull request merged**